compiler/prelude/prelude.go

```diff
@@ -293,6 +293,14 @@ var $clone = function(src, type) {
   return clone;
 };
 
+var $copyIntf = function(src) {
```

Comment:
I've never seen interfaces shortened to "intf". Instead of creating a novel way, let's reuse the same abbreviation this codebase (and Go codebase) already contains: "iface".

(https://github.com/golang/go/search?q=iface vs https://github.com/golang/go/search?q=intf)

Please update all places in this PR, not just this line.